framework/caching/Cache.php

```diff
@@ -71,6 +71,14 @@
      * implementations of the cache can not correctly save and retrieve data different from a string type.
      */
     public $serializer;
+    /**
+     * @var integer default ttl of cached value (the number of seconds in which the cached value will expire) -
```

Comment:
@var integer default duration in seconds before the cache will expire. Default value is 0, meaning infinity.